WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -365,6 +367,14 @@ private extension PointOfSaleAggregateModel {
                 let newPaymentState = PointOfSalePaymentState(from: paymentEvent,
                                                               using: presentationStyleDeterminerDependencies)
 
+                if case .card(.acceptingCard) = newPaymentState {
```

Comment:
👍 The fact that we can track all this in an aggregate model is nice. I agree that spreading it around other places of POS would become messy.